src/components/Header/Index.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react';
+import List from '../List/Index';
+import Button from '../Button/Index';
+import styles from './Index.module.scss';
+
+function Header() {
+  const [visibility, setVisibility] = useState(false);
+
+  const clickHandler = () => {
+    setVisibility(!visibility);
+  };
+
+  return (
+    <header className={styles.header}>
+      <nav className={styles.container}>
+        <Button clickHandler={clickHandler} />
```

Comment:
I suggest to use this code naming convention: 
`<Button onClick={handleClick}...`